pysnn/neuron.py

```diff
@@ -62,6 +64,7 @@ def update_trace(self, x):
 
     def forward(self, x):
         self.update_trace(x)
+        self.spikes.copy_(x)
```

Comment:
Now, `Input` is suited to take either spikes or scalar values to pass on, which is nice and flexible. But maybe we should check in some way or differentiate between the two?